### ISSUE TITLE
Authenticated failed error occurred

### ISSUE THREAD

**Issue by stmtstk** [ail-project/ail-framework#161]
Hi all,

I used the `ail-framework` branch named `KVROCKS` (https://github.com/ail-project/ail-framework/tree/KVROCKS) and tried some of its APIs, but I encountered an authentication failed error.

Upon inspecting the source code, I found the reason why this occurred:

https://github.com/ail-project/ail-framework/blob/ab7b2bdbabc7a19e979a4a597f7cd07bba76efa3/var/www/modules/restApi/Flask_restApi.py#L53

``` python
def verify_token(token):
    if len(token) != 41:
        return False

    if not check_token_format(token):
        return False

    return Users.exists_token(token)
```

This module checks the length of the API token, but the API token I received from the profile page in the web console (/settings/edit_profile) had a length of `55`.

As a result, the authentication error is always triggered.

Therefore, we need to modify the API token check logic or the way the API token is generated.

**Comment by adulau** [comment]
The bug is confirmed in the KVROCKS branch.

**Comment by stmtstk** [comment]
Thanks for your kind reply @adulau .
I am looking forward to the next release.

**Comment by Terrtia** [comment]
Hi @stmtstk ,

Thanks for the report ! 
AIL v5.0 has been released.
You can checkout the master branch.

Fixed in the new v5.0 release
